### PR TITLE
remove references to temporary data schema

### PR DESCRIPTION
**Pull request by pega-roska**
I'm not really sure what this was trying to get at before, but the temporary data schema has nothing to do with this part of the upgrade.

### CODE REVIEW

**Review by pega-talba — APPROVED** [review]

**Review by taz-pega-work — APPROVED** [review]

File: charts/pega/README.md [hunk 1]
```diff
@@ -805,7 +805,7 @@ Upgrade type    | Description
...
 `custom` |  Use this option for any upgrade in which you complete portions of the upgrade process in steps. Supported upgrade steps are: `enable_cluster_upgrade` `rules_migration` `rules_upgrade` `data_upgrade` `disable_cluster_upgrade`. To specify which steps to include in your custom upgrade, specify them in your pega.yaml file using the `upgrade.upgradeSteps` parameter.
-`out-of-place-rules` | Use this option to run an out-of-place upgrade of rules that runs against a migrated copy of the rules schema. This schema will become the rules schema after your upgrade is complete.
+`out-of-place-rules` | Use this option to migrate a copy of the rules to a new schema and run an out-of-place upgrade in that copied schema. This schema will become the rules schema after your upgrade is complete.
```
Comment (taz-pega-work) [inline charts/pega/README.md:808]: can you please clarify that this is the new rules schema, not just "a" schema?
Use this option to migrate a copy of the rules to a new rules schema and run an out-of-place upgrade of the rules in that copied schema. This schema will become the rules schema after your upgrade is complete.

**Review by taz-pega-work — COMMENTED** [review]
please make a slight tweak to this wording.

Also, please be sure to review the two runbooks I sent you, one for patching and one for ZDT updates to make sure that there's no offending or misleading statements in the runbooks. We made these changes and created the two runbooks around the same time.

File: charts/pega/README.md (hunk 1 above)
Comment (pega-roska) [reply charts/pega/README.md]: fixed

**Comment by pega-roska** [comment]
@taz-pega-work I think all your feedback should be addressed now.  Any further thoughts?

**Review by taz-pega-work — APPROVED** [review]
thanks so much for reviewing the runbooks @pega-roska !!!

ACE!

**Pull request merged**